text/66-self-managed-stack-sets.md

```diff
@@ -0,0 +1,502 @@
+---
+rfc pr: [#347](https://github.com/aws/aws-cdk-rfcs/pull/347)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/66
+---
+
+# CDK Self Managed Stack Set Support
+As a CDK user, I would like to define stack set infrastructure through native CDK constructs and create/update self-managed stack sets through CDK deploy command.
+
+* **Original Author(s):** @linsona
+* **Tracking Issue**: #66
+* **API Bar Raiser**: @skinny85
+
+## Working Backwards
+
+### CHANGELOG
+- feat(core): New Stack Set core type 
+- feat(assembly): New stack set artifcat type and schema
+- feat(cli): ```cdk deploy``` collects and creates/updates stack set 
+- feat(lib): API Calls/Logic for create/update stack sets
+
+### README
+The ```cdk.StackSet``` construct in CDK defines AWS resources in a stack set and stack set configuration/deployment preferences. This is different from the ```cdk.CfnStackSet``` construct which creates a CFN stack that deploys a CFN stack set. The ```cdk.StackSet``` construct creates the stack set directly if it does not exist, or updates a stack set and existing stack set instances while monitoring the stack set operation.
```

Comment:
Why the triple quotes?

Reply:
I can change to single or bold, just wanted to highlight

Reply:
Yes, let's just use single backticks for these. I don't think we need to bold them.